README.md

```diff
@@ -59,8 +59,17 @@ Example of `~/.ungitrc` configuration file to change default port and enable bug
 }
 ```
 
+
+SSH Authorization
+-----------------
+Ungit does run git commands in background and authentication will be asked continously when working with private repos.  Easiest way to prevent this is to set up ssh authorization and cache it.
+
+1. Follow github's [instruction](https://docs.github.com/en/github/authenticating-to-github/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent)
```

Comment:
1. Correctly capitalize GitHub
2. Remove language from URL to default to user language

```suggestion
1. Follow GitHub's [instructions](https://docs.github.com/github/authenticating-to-github/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent)
```